frb_codegen/src/parser/ty.rs

```diff
@@ -236,9 +236,24 @@ impl<'a> TypeParser<'a> {
                         other => IrType::Optional(IrTypeOptional::new(other)),
                     })
                 }
+                #[cfg(feature = "chrono")]
+                "DateTime" if matches!(*generic, SupportedInnerType::Path(SupportedPathType { ref ident, .. }) if ident == "Utc") => {
+                    Some(Delegate(IrTypeDelegate::Time(IrTypeTime::Utc)))
+                }
+
+                #[cfg(feature = "chrono")]
+                "DateTime" if matches!(*generic, SupportedInnerType::Path(SupportedPathType { ref ident, .. }) if  ident == "Local") => {
+                    Some(Delegate(IrTypeDelegate::Time(IrTypeTime::Local)))
+                }
```

Comment:
uber-nit: looks like we can extract common part here (e.g. "DateTime" and if matches). and a extra `match ident {local... utc...}

Reply:
resolved in [0a78151](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/0a781517c234523d0b5d995709ee51911c134ded)